test/runtests.jl

```diff
@@ -61,6 +61,11 @@ include(joinpath(examples_path, "kinsol_mkinTest.jl"))
 @test abs(minimum(residual)) < 1e-5
 end
 
+println("== handle tests")
+let
+@testset "Handle Tests" begin include("handle_tests.jl") end
```

Comment:
I would then remove the `println()` or think about avoiding typing the name of the test set in 3 places as it tends to be desynchronized.